src/contexts/auth/auth.provider.tsx

```diff
@@ -82,7 +82,6 @@ const AuthProvider = ({ children }: TAuthProviderProps) => {
 
   const updateCurrentLoginAccount = useCallback(
     (account: IUserLoginAccount) => {
-      setIsAuthorized(false);
```

Comment:
@agrim-deriv don't we need to update the authorize status anymore?

Reply:
yes, we still need to @shafin-deriv ; however, that should be done for the account switcher that will be present on the table for filtering purpose, as if we update the authorize status on the create api token page, as soon as we switch accounts it will reload the page and go back to the dashboard